ui-components/src/helpers/tableSummaries.tsx

```diff
@@ -1,81 +1,135 @@
 import * as React from "react"
 import { t } from "./translator"
-import { UnitSummary } from "@bloom-housing/backend-core/types"
+import { UnitSummary, UnitsSummary } from "@bloom-housing/backend-core/types"
 import { GroupedTableGroup } from "../tables/GroupedTable"
 
-export const unitSummariesTable = (summaries: UnitSummary[]) => {
-  const unitSummaries = summaries.map((unitSummary) => {
-    const minIncome =
-      unitSummary.minIncomeRange.min == unitSummary.minIncomeRange.max ? (
-        <strong>{unitSummary.minIncomeRange.min}</strong>
-      ) : (
-        <>
-          <strong>{unitSummary.minIncomeRange.min}</strong> {t("t.to")}{" "}
-          <strong>{unitSummary.minIncomeRange.max}</strong>
-        </>
-      )
+export const getSummaryRow = (
+  minIncomeRangeMin?: string,
+  minIncomeRangeMax?: string,
+  rentRangeMin?: string,
+  rentRangeMax?: string,
+  rentAsPercentIncomeRangeMin?: string,
+  rentAsPercentIncomeRangeMax?: string,
+  totalAvailable?: number,
+  totalCount?: number,
+  unitTypeName?: string
+) => {
+  let minIncome = <></>
+  if (minIncomeRangeMin == undefined) {
+    //TODO(#354): figure out what to display when there's no data
```

Comment:
nit: 345 instead of 354 (here and below)

Reply:
Updated